OpenRailAssociation/osrd
# front: stdcm: update simulation progress button design

close #9677 
fix #9322 

To test popup behavior : 
- Animation is not on sketch yet and its duration will be validated with @thibautsailly on November 25th.
When launching a simulation, the launch button should fade out while the simulation progress popup should fade in from the bottom.
- The popup should be sticky to the bottom if it doesn't fit below the form and be fixed to the top when scrolling far bottom (if you have already a simulation in the results, you can toggle the op list to increase the height and test the behavior)
